src/model/user.py

```diff
@@ -41,6 +54,24 @@ class Meta:
 user_schemas = UserSchema(many=True)
 
 
+class RoleSchema(ModelSchema):
```

Comment:
Clean! We can move all role-related stuff to its own schema file to make things more neat